draft-ietf-acme-acme.md

```diff
@@ -517,6 +517,11 @@ to errors, this document defines the following standard tokens for use in the
 | tls             | The server experienced a TLS error during DV             |
 | unauthorized    | The client lacks sufficient authorization                |
 | unknownHost     | The server could not resolve a domain name               |
+| rateLimited     | The request exceeds a rate limit                         |
+
+This list is not exhaustive. The server MAY return errors whose "type"
```

Comment:
I think that server should not return anything in `urn:acme:` subspace unless it's registered with the spec